src/Governance.sol

```diff
@@ -685,11 +687,6 @@ contract Governance is MultiDelegateCall, UserProxyFactory, ReentrancyGuard, Own
         int256[] memory _deltaOffsetVotes,
         int256[] memory _deltaOffsetVetos
     ) internal {
-        require(
```

Comment:
Maybe we can put a “@dev” comment in the header of this function saying that it  assumes all param array lengths are equal.